spring-boot-modules/spring-boot-featureFlag-Unleash/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-boot-featureFlag-Unleash</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>spring-boot-featureFlag-Unleash</name>
+    <description>Demo project for Spring Boot FeatureFlag with Unleash</description>
+
+    <parent>
+        <groupId>com.baeldung.spring-boot-modules</groupId>
```

Comment:
We also need to add the module to the parent pom.

Reply:
Done.